src/profile/sagas.js

```diff
@@ -135,7 +135,7 @@ export function* handleSaveProfile(action) {
     // The preferences draft is valid if the server didn't complain, so
     // pass it through directly.
     yield put(saveProfileSuccess(accountResult, preferencesResult));
-    yield delay(300);
+    yield delay(1000);
```

Comment:
Might be nice to make this a constant and / or have it built into the confirmation component.